output.tf

```diff
@@ -1,9 +1,9 @@
 output "public_address" {
```

Comment:
Shouldn't this return the public address from the external Application Gateway?

Reply:
For now, after latest changes if disable_agw is true it won't return anything. I can try to make it return the external_fdqn for clarity if you think it would be more convenient with the context path included perhaps? 
Please share your thoughts and I will change it or leave it as it is.